testslide/cli.py

```diff
@@ -196,6 +196,11 @@ def _build_parser(self, disable_test_files):
             action="store_true",
             help="Only executed focused examples, or all if none focused",
         )
+        parser.add_argument(
+            "--fail-if-focused",
+            action="store_true",
+            help="Raise an error if an example is focused",
```

Comment:
Worth explaining the rational perhaps: *Raise an error if an example is focused. Useful when running tests at a continuous integration environment.*.